src/sentry/models/notificationsetting.py

```diff
@@ -59,8 +59,8 @@ def provider_str(self) -> str:
         null=False,
     )
     scope_identifier = BoundedBigIntegerField(null=False)
-    target = FlexibleForeignKey(
-        "sentry.Actor", db_index=True, unique=False, null=False, on_delete=models.CASCADE
+    target_id = HybridCloudForeignKey(
```

Comment:
Why is this changing to be nullable? There shouldn't ever be a notification setting that's null, but I'm guessing this has to do with the migration plan?

Reply:
Yeah, wondering if that's a mistake? We aren't changing this at the db level here since the migration is state only.